cmake/MeasurementKit.cmake

```diff
@@ -0,0 +1,19 @@
+## measurement_kit
+
+file(
+  GLOB
+  MK_PROGRAM_SOURCES
+  "src/measurement_kit/*.cpp"
+  "src/measurement_kit/*/*.cpp"
```

Comment:
Maybe you also want this to be prefixed with `MEASUREMENTKIT_ROOT`. I didn't set it since I don't need to compile the binary, but for consistency this could be changed too.

Reply:
Yes, please, do that

Reply:
ping @hellais